fixtures/attribute-behavior/AttributeTableSnapshot.md

```diff
@@ -4827,20 +4827,20 @@
 | Test Case | Flags | Result |
 | --- | --- | --- |
 | `hidden=(string)`| (changed)| `<boolean: true>` |
-| `hidden=(empty string)`| (initial)| `<boolean: false>` |
+| `hidden=(empty string)`| (changed)| `<boolean: true>` |
```

Comment:
Similar issue as https://github.com/facebook/react/pull/24730: Should this be behind a flag?